src/actions/accountEntryActions.js

```diff
@@ -0,0 +1,74 @@
+import { trimObjValues } from "../util/utilFunctions.js";
+
+export const signup = newUserData => {
+  return new Promise((resolve, reject) => {
+    const alreadyFoundError = new Error("usernameAlreadyExists");
```

Comment:
Optional improvement: Move this to a different file called `errors.js`. Then you can reference it outside of this promise scope, for example if you wanted to reference it in unit tests.